### PR TITLE
Add fetch plus plus

### PR DESCRIPTION
**Pull request by gndelia**
This PR adds the `fetch-plus-plus` package, as well as enabling Github Actions to check formatting and run tests (these should start running once merged)

### Process checklist

- [x] Manual tests passed or not applicable.
- [x] Automated tests added or not applicable.
- [ ] Documentation updated or not required.

### Metrics

Actual effort: 1h

### CODE REVIEW

**Review by gabmontes — CHANGES_REQUESTED** [review]
Please add a comprehensive `README.md` to the package.

**Comment by gndelia** [comment]
> Please add a comprehensive `README.md` to the package.

@gabmontes  `README.md` added [e9b1f0e](https://github.com/bloq/js-packages/pull/5/commits/e9b1f0ea2b4d38b3398a91e148d0ee2c1f64573b) - and I also added the badges for `npm` here [981761e](https://github.com/bloq/js-packages/pull/5/commits/981761e16ba9dfa8ed7167cd168b62771c69c793) 😄

File: packages/fetch-plus-plus/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "license": "MIT",
+  "author": {
+    "name": "Gonzalo D'Elia",
+    "email": "gonzalo@bloq.com"
+  },
+  "main": "src/index.js",
+  "dependencies": {
+    "http-errors": "2.0.0",
+    "isomorphic-fetch": "3.0.0",
+    "p-retry": "4.6.1"
```
Comment (gabmontes) [inline packages/fetch-plus-plus/package.json:22]: Packages should not have pinned versions. Let's allow minor and patched dependencies to be used too.

```suggestion
    "http-errors": "^2.0.0",
    "isomorphic-fetch": "^3.0.0",
    "p-retry": "^4.6.1"
```

**Review by gabmontes — CHANGES_REQUESTED** [review]

File: packages/fetch-plus-plus/package.json (hunk 1 above)
Comment (gndelia) [reply packages/fetch-plus-plus/package.json]: Versions unpinned here [2a572e3](https://github.com/bloq/js-packages/pull/5/commits/2a572e33bba0998b024a697fd0a52e0e6bb6efe0)

**Pull request merged**